docs/source/index.rst

```diff
@@ -0,0 +1,79 @@
+.. raw:: html
+
+  <h1>Welcome</h1>
+
+  <h2 style="margin-bottom: 10 px">Start Here</h2>
+
+  <div style='column-count: 2; margin-left: 30px;'>
+      <p class="biglink"><a class="biglink" href="{{ pathto("content-quickstart") }}">Jupyter Notebook Quickstart</a><br/>
+         <span class="linkdescr">Try the notebook</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("architecture/content-architecture") }}">Architecture</a><br/>
+         <span class="linkdescr">What is Jupyter?</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("use-cases/content-user") }}">Narratives and Use Cases</a><br/>
```

Comment:
Still I have the `{{ pathto("use-cases/content-user") }}` that are not interpreted. 
Either we should have a custom sphinx directive that understand that, or we just render content through normal rst, and just change css.